File: algorithm/src/main/java/baekjoon/p02153/Main.java
package baekjoon.p02153;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println(solve(sc.next()));
    }

    private static String solve(String input) {
        int sum = 0;
        for (char ch : input.toCharArray()) {
            if (ch >= 'a' && ch <= 'z') {
                sum += (ch - 'a' + 1);
            } else {
                sum += (ch - 'A' + 27);
            }
        }

        if (prime(sum)) {
            return "It is a prime word.";
        } else {
            return "It is not a prime word.";
        }
    }

    private static boolean prime(int n) {
        for (int i = 2; i < n; i++) {
            if (n % i == 0) {
                return false;
            }
        }

        return true;
    }

}

File: algorithm/src/main/java/baekjoon/p02153/Main.kt
package baekjoon.p02153

fun main(args: Array<String>) {
    println(solve(readLine()!!))
}

private fun solve(input: String): String {
    val sum = input.sumBy {
        if (it.isLowerCase()) {
            it - 'a' + 1
        } else {
            it - 'A' + 27
        }
    }

    return when (prime(sum)) {
        true -> "It is a prime word."
        false -> "It is not a prime word."
    }
}

private fun prime(n: Int): Boolean {
    for (i in 2 until n) {
        if (n % i == 0) {
            return false
        }
    }

    return true
}